logicalplan/distribute.go

```diff
@@ -129,6 +129,7 @@ var distributiveAggregations = map[parser.ItemType]struct{}{
 	parser.SUM:     {},
 	parser.MIN:     {},
 	parser.MAX:     {},
+	parser.AVG:     {},
```

Comment:
I think this might be harmless, but technically its not true that AVG is distributive right? We work around it by rewriting it to sum/count in a different optimizer so that we never have to see avg in this optimizer if i understand correctly. Adding this here seems like it could be a subtle bug waiting to happen

Reply:
I think you are right! 

I will try to fix this next week, since we have a separate optimizer that rewrites `avg`, we should remove it from this list.